youtube_dl/extractor/tvanouvelles.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class TVANouvellesIE(InfoExtractor):
+    _VALID_URL = r'https?://(www\.|)tvanouvelles\.(ca|com|qc)/.*.?/(?P<id>[^/]+)'
```

Comment:
I am not very familiar to Python regex. I just wanted to give a possibility that it might have the www. or not. for .*.? was to ignore everything until you see the last /. The reason is because this can also be the URL: [http://www.tvanouvelles.ca/2016/09/11/les-etats-unis-commemorent-le-traumatisme-du-11-septembre](http://www.tvanouvelles.ca/2016/09/11/les-etats-unis-commemorent-le-traumatisme-du-11-septembre).